model_compression_toolkit/quantizers_infrastructure/common/constants.py

```diff
@@ -15,7 +15,8 @@
 
 IS_WEIGHTS = "is_weights"
 IS_ACTIVATIONS = "is_activations"
-
+ACTIVATION_QUANTIZERS = "activation_quantizers"
+WEIGHT_QUANTIZERS = "weight_quantizer"
```

Comment:
Maybe change to "WEIGHT**S**" (and "weight**s**") so it would be consistent with the rest of the constants (and generally with the rest of the code)